geopandas/tests/test_plotting.py

```diff
@@ -286,6 +288,16 @@ def test_empty_plot(self):
             ax = df.plot()
         assert len(ax.collections) == 0
 
+        if compat.USE_PYGEOS:
+            s = GeoSeries([wkt.loads("POLYGON EMPTY")])
```

Comment:
Can you make these empty + normal geometry? So one of them plots, the other does not.